test/bonjour.js

```diff
@@ -135,6 +151,7 @@ test('bonjour.findOne - emitter', function (bonjour, t) {
     var browser = bonjour.findOne({ type: 'test' })
     browser.on('up', function (s) {
       t.equal(s.name, 'Emitter')
+      browser.stop()
```

Comment:
no, it was one of my tentative to fix a bug that I didn't clear ...  I can remove it if you want me to.  
for whatever reason, the [buildServicesFor](https://github.com/watson/bonjour/blob/master/lib/browser.js#L134) function acted weirdly during the test; it was pushing duplicate addresses to the addresses field.
I resolved it, but I can't really explain why it was doing that.